app/src/main/java/org/sopt/sample/week1/signup/SignUpPolicy.kt

```diff
@@ -0,0 +1,17 @@
+package org.sopt.sample.week1.signup
+
+import org.sopt.sample.databinding.ActivitySignUpBinding
+
+class SignUpPolicy(val idMin: Int, val idMax: Int, val pwMin: Int, val pwMax: Int) {
+
+    fun passwordInvalid(binding: ActivitySignUpBinding) =
+        binding.idSucceed() && !binding.passwordSucceed()
+
+    fun idInvalid(binding: ActivitySignUpBinding) = !binding.idSucceed()
```

Comment:
```suggestion
    fun isIdInvalid(binding: ActivitySignUpBinding) = !binding.idSucceed()
```

판별하는 함수이니 is를 접두사로 붙이는건 어떤가요?